mandos-go/json/write/writeAccount.go

```diff
@@ -41,6 +41,9 @@ func AccountsToOJ(accounts []*mj.Account) oj.OJsonObject {
 		if len(account.Owner.Value) > 0 {
 			acctOJ.Put("owner", bytesFromStringToOJ(account.Owner))
 		}
+		if len(account.DeveloperReward.Original) > 0 {
+			acctOJ.Put("developerRewards", bigIntToOJ(account.DeveloperReward))
```

Comment:
For another time: would it be possible to refactor this using the generics in Go 1.18?

Reply:
we're still using go 1.13